### PR TITLE
B-18268

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Allow TOO to edit PPM Shipment Card
## Verification Steps for the Author

These are to be checked by the author.

- [x] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [x] Has the branch been pulled in and checked out?
- [x] Have the BL acceptance criteria been met for this change?
- [x] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office app
2. Login as a TOO
3. Find a move with a PPM shipment
4. Click "Edit Shipment" under the ppm shipment
5. 

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

Screenshot 2023-12-26 at 2 19 10 PM

Screenshot 2023-12-26 at 2 19 25 PM

### CODE REVIEW

File: src/pages/Office/index.jsx [hunk 1]
```diff
@@ -295,6 +295,15 @@ export class OfficeApp extends Component {
                         </PrivateRoute>
                       }
                     />
+                    <Route
```
Comment (danieljordan-caci) [inline src/pages/Office/index.jsx:298]: Could you update `index.test.jsx` in this same folder to reflect this routing change? The second to last `it.each` test should be a one liner update to account for the new route and that it generates header content.

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

File: src/pages/Office/index.jsx (hunk 1 above)
Comment (deandreJones) [reply src/pages/Office/index.jsx:298]: added

**Review by danieljordan-caci — APPROVED** [review]
L
    G
        T
            M

**Review by TevinAdams — APPROVED** [review]
Tests passing, Functionality works, Functionality meets ACs, and reviewed code.

**Review by cameroncaci — APPROVED** [review]

**Pull request merged**